iocore/dns/SRV.h

```diff
@@ -50,9 +49,8 @@ operator<(const SRV &left, const SRV &right)
 }
 
 struct SRVHosts {
-  unsigned int srv_host_count   = 0;
   unsigned int srv_hosts_length = 0;
-  SRV hosts[HOST_DB_MAX_ROUND_ROBIN_INFO];
+  std::vector<SRV> hosts;
```

Comment:
I see, the srv_host_length is the string length of all hosts. If so, I guess that's an ok optimization, I'd probably examine how often it's used, and if so, how expensive it is to iterator over it to get the lengths.

But if you decide to leave it, I'm fine with that.